tutorbackup/plugin.py

```diff
@@ -2,7 +2,7 @@
 from .__about__ import __version__
 from glob import glob
 import os
-import pkg_resources
+import importlib_resources
```

Comment:
We don't add any dependency on `importlib_resources` in this change, and we don't need to, because we do depend on `tutor`, which as of https://github.com/overhangio/tutor/commit/d99b2feeb30fa772dca48345c8992b4eafdeac21 does have that dependency.

As explained in https://github.com/overhangio/tutor/issues/966#issuecomment-1938681102, Tutor will drop that dependency once it no longer wants to support Python 3.8. At that point, this `import` statement will also break, and we will replace it with 

```python
from importlib import resources as importlib_resources
```

Perhaps we want to add a comment to that effect _now,_ as a benefit to our future selves?